CesiumJsonReader/src/JsonObjectJsonHandler.cpp

```diff
@@ -62,7 +62,15 @@ IJsonHandler* JsonObjectJsonHandler::readDouble(double d) {
 }
 
 IJsonHandler* JsonObjectJsonHandler::readString(const std::string_view& str) {
-  *this->_stack.back() = std::string(str);
+  CesiumUtility::JsonValue& current = *this->_stack.back();
+  CesiumUtility::JsonValue::Array* pArray =
+      std::get_if<CesiumUtility::JsonValue::Array>(&current.value);
+  if (pArray) {
+    pArray->emplace_back(std::string(str));
+  } else {
+    current = std::string(str);
+  }
+
```

Comment:
This could have also been implemented as `addOrReplace(*this->_stack.back(), std::string(str));`

But I couldn't think of a clean way to get around the extra string copies